site/events.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: Upcoming Events
+permalink: /events/
+---
+
+[Archived Events](/past-events/)
+
+{% assign current_events = site.events | where_exp: "event", "event.date >= site.time" %}
```

Comment:
i wonder if we'll have to change how the sorting works here, but i think it works the way we want. i can't quite remember.

Reply:
Upon testing, I believe this is naturally sorted the way we want it for upcoming events.  This means it's likely a problem for archived events.

I figured it's an easy change once it becomes an issue.